atomic_reactor/schemas/container.json

```diff
@@ -28,7 +28,34 @@
       "additionalProperties": false
     },
     "compose": {
-      "type": "object"
+      "type": "object",
+      "properties": {
+        "packages": {
+          "description": "names of RPMs to include",
+          "type": ["array", "null"],
+          "items": {
+              "type": "string"
+          }
+        },
+        "pulp_repos": {
+          "description": "whether to build pulp composes",
+          "type": "boolean"
+        },
+        "modules": {
+          "description": "names of modules to include",
+          "type": ["array", "null"],
+          "items": {
+              "type": "string"
+          }
+        },
+        "signing_intent": {
+          "type": ["string", "null"]
```

Comment:
I think it has to be a string doesn't it?

Reply:
Fixed.

Reply:
Helps if I push the fix.